### PR TITLE
fix(node): CI always evaluated as true

### LINKED ISSUE #352: process.env.CI is always evaluated to true
process.env.CI **is always** evaluated to true

```
> process.env.CI ? ['--frozen-lockfile'] : []
[]
> process.env.CI = 'false'
'false'
> process.env.CI ? ['--frozen-lockfile'] : []
[ '--frozen-lockfile' ]
> process.env.CI = 'true'
'true'
> process.env.CI ? ['--frozen-lockfile'] : []
[ '--frozen-lockfile' ]
> process.env.CI = 1
1
> process.env.CI ? ['--frozen-lockfile'] : []
[ '--frozen-lockfile' ]
> process.env.CI = 0
0
> process.env.CI ? ['--frozen-lockfile'] : []
[ '--frozen-lockfile' ]
```

https://github.com/projen/projen/blob/5776b6a26b3a4a0e5366541a1dea84d24f437b33/src/node-project.ts#L1304

### PR DESCRIPTION
**Pull request by Chriscbr**
Fixes #352

### CODE REVIEW

File: test/util.test.ts [hunk 1]
```diff
@@ -73,3 +73,12 @@ describe('decamelizeRecursively', () => {
...
 });
 
+test('isTruthy', () => {
+  expect(isTruthy(undefined)).toEqual(false);
+  expect(isTruthy('false')).toEqual(false);
+  expect(isTruthy('0')).toEqual(false);
+  expect(isTruthy('null')).toEqual(false);
+  expect(isTruthy('true')).toEqual(true);
+  expect(isTruthy('1')).toEqual(true);
+  expect(isTruthy('enabled')).toEqual(true);
```
Comment (tlakomy) [inline test/util.test.ts:83]: I'd recommend adding a test case for an empty string as well

Comment (Chriscbr) [reply test/util.test.ts:84]: good point!

**Review by eladb — APPROVED** [review]
